packages/jsii-python-runtime/src/jsii/_kernel/__init__.py

```diff
@@ -120,6 +123,39 @@ def _make_reference_for_native(kernel, d):
         return d
 
 
+def _handle_callback(kernel, callback):
+    # need to handle get, set requests here as well as invoke requests
+    if callback.invoke:
+        obj = _reference_map.resolve_id(callback.invoke.objref.ref)
+        method = getattr(obj, callback.cookie)
+        return method(*callback.invoke.args)
+    elif callback.get:
+        obj = _reference_map.resolve_id(callback.get.objref.ref)
+        return getattr(obj, callback.cookie)
+    elif callback.set:
+        obj = _reference_map.resolve_id(callback.set.objref.ref)
+        return setattr(obj, callback.cookie, callback.set.value)
+    else:
+        raise AttributeError
```

Comment:
Do we want this to be a ``JSIIError`` or something? Or at the very least have a descriptive message? Ideally we should *never* hit this unless the JSII adds another type of callback, so it's basically just asserting a known assumption about the world.